src/analyze_shapes.cc

```diff
@@ -0,0 +1,55 @@
+#include "motis/analyze_shapes.h"
+
+#include "utl/verify.h"
+
+#include "fmt/base.h"
+
+#include "nigiri/shapes_storage.h"
+#include "nigiri/types.h"
+
+#include "motis/tag_lookup.h"
+
+namespace motis {
+
+bool analyze_shape(nigiri::shapes_storage const& shapes,
+                   nigiri::trip_idx_t const& trip_idx) {
+  auto const offset_idx = shapes.trip_offset_indices_[trip_idx].second;
+  if (offset_idx == nigiri::shape_offset_idx_t::invalid()) {
+    fmt::println("No shape offsets for '{}'", trip_idx);
```

Comment:
is the trip_idx really helping the user? user works with trip ids from the URL, right?